scripts/tests/test_generate_sitemap.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+
```

Comment:
No, it should be running with travis since it is this script runs nightly via celery. Per Steve, move it to the `osf_tests` folder.